sdk/java/src/main/java/com/chain/api/Account.java

```diff
@@ -253,4 +253,94 @@ public Builder setTags(Map<String, Object> tags) {
       return this;
     }
   }
+
+  /**
+   * Use this class to create a {@link Receiver} under an account.
+   */
+  public static class ReceiverBuilder {
+    @SerializedName("account_alias")
+    public String accountAlias;
+
+    @SerializedName("account_id")
+    public String accountId;
+
+    @SerializedName("expires_at")
+    public Date expiresAt;
+
+    /**
+     * Specifies the account under which the receiver is created. You must use
+     * this method or @{link ReceiverBuilder#setAccountId}, but not both.
+     *
+     * @param alias The unique alias of the account.
+     * @return The updated ReceiverBuilder object.
```

Comment:
Really great question, I was wondering about myself as I was literally going through and capitalizing first letters, adding articles, and adding periods. It's kind of all over the place, so I just started doing what made sense to me in this PR. It would be good to codify that at some point.

FWIW, what do you think your preference would be?